codecs/h264_packet_test.go

```diff
@@ -0,0 +1,48 @@
+package codecs
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNextNALU(t *testing.T) {
+	assert := assert.New(t)
+
+	input := []byte{
+		0x00, 0x00, 0x00, 0x01, 0x67, 0x42, 0x00, 0x1f, 0x01,
+		0x00, 0x00, 0x01, 0x80, 0x00, 0x00, 0x03, 0x00, 0x7a,
+		0x00, 0x00, 0x00, 0x01, 0xab, 0xbc, 0xef,
+	}
+
+	// The first NALU is empty to trim off the start code.
+	nalu, input := nextNALU(input)
+	assert.Len(nalu, 0)
```

Comment:
Personal opinion: I don't think test helpers are that helpful.

```go
if len(nalu) != 0 {
  t.Fatalf("nextNalu() returned empty result")
}
```

Is more readable and doesn't need an extra dependency.